diff --git a/index.js b/index.js
index d695233eda4652b4c554a0967970295d7b9cfdaa..039a7bfab4dc0ec66cea429bf53031fd6301f2c7 100644
--- a/index.js
+++ b/index.js
@@ -9,10 +9,13 @@ module.exports = inquire;
  */
 function inquire(moduleName) {
   try {
-    if (typeof require !== "function") {
+    var req = Function(
+      "return typeof require === 'function' ? require : null;"
+    )();
+    if (!req) {
       return null;
     }
-    var mod = require(moduleName);
+    var mod = req(moduleName);
     if (mod && (mod.length || Object.keys(mod).length)) return mod;
     return null;
   } catch (err) {
